<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QWidget" name="horizontalWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="verticalWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTreeView" name="treeView">
            <property name="minimumSize">
             <size>
              <width>180</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>180</width>
              <height>16777215</height>
             </size>
            </property>
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="minimumSize">
             <size>
              <width>180</width>
              <height>128</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>180</width>
              <height>180</height>
             </size>
            </property>
            <property name="title">
             <string>Фигура</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QScrollArea" name="scrollArea">
               <property name="widgetResizable">
                <bool>true</bool>
               </property>
               <widget class="QWidget" name="scrollAreaWidgetContents">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>176</width>
                  <height>163</height>
                 </rect>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                   <property name="spacing">
                    <number>3</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_4">
                     <item>
                      <widget class="QLabel" name="label">
                       <property name="text">
                        <string>Ширина</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QSpinBox" name="spin_width">
                       <property name="maximum">
                        <number>100</number>
                       </property>
                       <property name="value">
                        <number>80</number>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_5">
                     <item>
                      <widget class="QLabel" name="label_2">
                       <property name="text">
                        <string>Высота</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QSpinBox" name="spin_heigth">
                       <property name="maximum">
                        <number>100</number>
                       </property>
                       <property name="value">
                        <number>40</number>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_6">
                     <item>
                      <widget class="QLabel" name="label_3">
                       <property name="text">
                        <string>Радиус</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QSpinBox" name="spinBox_radius">
                       <property name="maximum">
                        <number>100</number>
                       </property>
                       <property name="value">
                        <number>15</number>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout">
                     <item>
                      <widget class="QLabel" name="label_4">
                       <property name="text">
                        <string>Перо</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QComboBox" name="combo_pen">
                       <item>
                        <property name="text">
                         <string>Черный</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>Белый</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>Красный</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>Зеленый</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>Синий</string>
                        </property>
                       </item>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_7">
                     <item>
                      <widget class="QLabel" name="label_5">
                       <property name="text">
                        <string>Заливка</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QComboBox" name="combo_back">
                       <property name="currentIndex">
                        <number>1</number>
                       </property>
                       <item>
                        <property name="text">
                         <string>Черный</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>Без цвета</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>Красный</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>Зеленый</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>Синий</string>
                        </property>
                       </item>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="lineWidth">
          <number>2</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MousedView" name="graphicsView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_about"/>
    <addaction name="action_exit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action_about">
   <property name="text">
    <string>Помощь</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MousedView</class>
   <extends>QGraphicsView</extends>
   <header>mousedrawingview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
